### DISCUSSION TITLE
The src-tauri folder size is too large

### DISCUSSION THREAD

**Discussion by sanbei101** [tauri-apps/tauri#10237] [General]
**这个文件夹在下载依赖并且构建编译之后往往能达到3个GB以上,大家也有这个问题吗**

> **This folder often reaches more than 3GB after downloading dependencies and building and compiling, does anyone else have this problem?**

**Comment by FabianLars** [comment]
2 to 3gb is to be expected. This is the drawback of using Rust. Or i guess, it's more of a tradeoff, you have a large target folder and long compile times but your users have small and fast apps 🤷 

You can regularly delete the `target` folder if it gets too large, just keep in mind that the next time you build your app afterwards it will take a while longer to rebuild everything.

**Comment by mollerzhu** [comment]
我6个G

**Comment by alexlee85** [comment]
just run `cargo clean` in src-tauri folder after every release
